worker/src/main/resources/application.properties

```diff
@@ -52,7 +52,7 @@ coverage.update.initial.delay=1
 ## ---------------------------------------------------------------------------- STUCK JOB
 ## -- run every hour
 stuck.job.cron.schedule=0 0 * * * ?
-stuck.job.cancel.threshold=24
+stuck.job.cancel.threshold=72
```

Comment:
Do we want to increase the stuck job timeout here? I thought it was unrelated to the file deletion timeout.

Reply:
It is, and maybe 72 is too much, just thought it might be a possibility for longer jobs. Granted, the entire completion isn't 72 hours, just the time to complete and then download so maybe I can make it somewhere in the middle.

Reply:
I'll make it 36.

Reply:
Well the file deletion service only deletes files 72 hours after the job completes. You also can't download any files until after the job completes so the stuck time is kind of independent. Is stuck time sensitive to % complete?

Reply:
No, just trying to hedge against the possibility of a long job.